ssm_agent_endpoint_sg.tf

```diff
@@ -0,0 +1,23 @@
+# Security group for the VPC interface endpoints in the private subnet
+# that are required by the SSM agent
+resource "aws_security_group" "ssm_agent_endpoint" {
+  provider = aws.sharedservicesprovisionaccount
+
+  vpc_id = aws_vpc.the_vpc.id
+
+  tags = {
+    Name = "SSM agent endpoints"
```

Comment:
Is there a reason the resource name is singular but the tag name is plural?

Reply:
The security group is applied to multiple VPC endpoints, since access to multiple VPC endpoints is required to make use of the SSM Agent.